_data/tools.yml

```diff
@@ -1384,6 +1384,17 @@
     or code from descriptions!
   v2: true
 
+- name: docs
+  category:
+    - parser
+    - converters
+    - sdk
+  link: https://github.com/go-oas/docs
+  github: https://github.com/go-oas/docs
+  description: A modern alternative to `go-swagger`. Generation and parsing of OpenAPI Specs, depending on the usage. Requires traction for further development.
```

Comment:
I am not sure this is the best description. It's advertising that this project is dead until people use it?

Reply:
The comment was updated per your advice. Worth merging? @daveshanley